src/solution.jl

```diff
@@ -13,10 +14,21 @@ mutable struct Solution
         # M3_ij is the number of subsequences in which the number of
         # cars that require oj is greater than or equal to p(oj)
 
-    Solution(nC,nO) = new(
-        zeros(Int,nC),
+    Solution(nC::Int,nO::Int) = new(
+        collect(1:nC),
```

Comment:
I think `collect(1:nC)` is too much for an empty solution. Initialization with `zeros` could be an efficient "default value" when we will implement a greedy algorithm.

Reply:
I think a default value should be a solution, and zeros is not. However I can accept to change if other people agree with you.

Reply:
I agree with @Jonathan-Fontaine44, zeros is not a good solution. For an empty solution we could go for `nothing` as well.